opm/output/eclipse/summaryComparator.hpp

```diff
@@ -0,0 +1,94 @@
+/*
+   Copyright 2016 Statoil ASA.
+
+   This file is part of the Open Porous Media project (OPM).
+
+   OPM is free software: you can redistribute it and/or modify it under 
+   the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   OPM is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <iostream>
+#include <vector>
+#include <algorithm>
+#include <string>
+
+
+struct stringlist_struct;
+typedef struct stringlist_struct stringlist_type;
+struct ecl_sum_struct;
+typedef struct ecl_sum_struct ecl_sum_type;
+
+
+
+struct Deviation {
+  double abs; 
+  double rel; 
+};
+
+
+class SummaryComparator {
```

Comment:
this is a public interface. i expect doxy on at least the part exposed to the world.